src/components/classrooms/ClassroomEditor.jsx

```diff
@@ -175,15 +176,26 @@ const ClassroomEditor = (props) => {
             {props.assignmentsStatus === ASSIGNMENTS_STATUS.SUCCESS &&
             props.assignments[props.selectedClassroom.id] &&
             props.assignments[props.selectedClassroom.id].length > 0 &&
-              students.map(student =>
-                (<TableRow className="manager-table__row-data" key={`classroom-student-${student.id}`}>
+              students.map((student) => {
+                const galaxyAssignment = props.assignments[props.selectedClassroom.id].filter(
+                  assignment => assignment.name === i2aAssignmentNames.first);
+                const hubbleAssignment = props.assignments[props.selectedClassroom.id].filter(
+                  assignment => assignment.name === i2aAssignmentNames.second);
+
+                // Why are the ids in the student_user_id property numbers?!?!?!
+                const galaxyStudentData = galaxyAssignment[0].studentAssignmentsData.filter(
+                  data => data.attributes.student_user_id.toString() === student.id);
+                const hubbleStudentData = hubbleAssignment[0].studentAssignmentsData.filter(
+                  data => data.attributes.student_user_id.toString() === student.id);
+
+                return (<TableRow className="manager-table__row-data" key={`classroom-student-${student.id}`}>
                   <td headers="student-name">
                     {(student.zooniverseDisplayName && student.zooniverseDisplayName.length > 0)
                       ? <span>{student.zooniverseDisplayName}</span>
                       : <span className="secondary">{student.zooniverseLogin}</span> }
                   </td>
-                  <td headers="assignment-galaxy">{Math.floor(Math.random() * 100)}%</td>
-                  <td headers="assignment-hubble">{Math.floor(Math.random() * 100)}%</td>
+                  <td headers="assignment-galaxy">{(galaxyStudentData[0].attributes.classifications_count / (+galaxyAssignment[0].metadata.classifications_target)) * 100}%</td>
```

Comment:
1. Is `attributes.classifications_count` always a number, not a string?
  - (Also note conversation with Amy on Slack on #education-api at 11am 15 Nov 2017 Chicago, where some comments were made about converting numbers to string as a standard... I'm not sure if there's going to be some blanket update to the number values that may affect this.)
2. I was going to ask if `classifications_target` would ever be 0, but then I realised that... maybe we don't need to worry too much, since with JS, if you want to display some nonsensical value like `123 / 0` or `123 / undefined`, the code will just shrug and display NaN or Infinity, instead of crashing. I'll doublecheck this, but if this interpretation is correct, then it's _safe enough_ to display data like this without having to worry about the divide-by-0 cases.